server/src/Domain/HumanResource/User/UserAdministrative.entity.ts

```diff
@@ -57,7 +60,8 @@ export class UserAdministrative {
     workingTime: WorkingTimeType,
     joiningDate: string,
     leavingDate?: string,
-    transportFee?: number
+    transportFee?: number,
+    sustainableMobilityFee?: number
```

Comment:
Si on applique le commentaire précédent

```suggestion
    sustainableMobilityFee = 0
```